datamuse/datamuse.py

```diff
@@ -0,0 +1,47 @@
+from urllib import urlencode
+import urllib2
+import json
+
+class Datamuse:
+    def __init__(self):
+        self.API_ENDPOINT = 'https://api.datamuse.com/words'
+
+    def make_api_request(self, params):
+        request = urllib2.urlopen(self.API_ENDPOINT + '?' + params)
+        return json.loads(req.read().decode("utf-8"))
+
+    def get_means_like(self, phrase):
```

Comment:
I think my preferred way to do this is to have a single function that accepts either a `**kwargs` or a big list of optional params (I'm not sure what is the more accepted style), then goes through each of the things specified there to add them to the request. Then we can have the expected params have more readable names than the secret ones.

Some examples of what this might look like to call:
`dm.get_translation(means_like="sandwich")`
`dm.get_translation(rhymes_with="sandwich")`
`dm.get_translation(means_like="sandwich", rhymes_with="bub")`